pkg/odo/cli/application/delete.go

```diff
@@ -80,7 +80,7 @@ func (o *DeleteOptions) Run() (err error) {
 		return o.appClient.Delete(o.appName)
 	}
 
-	// Print App Information which will be deleted
+	// Printf App Information which will be deleted
```

Comment:
Why "Printf"? The `printAppInfo` function is printing AFAICS.

Reply:
Damn! I think this was because of renaming. I renamed the function from Print to Printf, and looks like it renamed functions it shouldn't have. Thanks for catching them.